Update Set-CsCallQueue.md (#10548)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsCallQueue.md

Accept wildcard characters: False
```

### -AuthorizedUsers

> This is a list of user GUIDs who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 

Reviewer: ```suggestion
This is a list of GUIDs of users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with four dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

> This is a list of user GUIDs who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 

Reviewer 2: ```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue.  The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

> This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 

Reviewer 2: ```suggestion
This is a list of GUIDs for users who are authorized to make changes to this call queue. The users must also have a TeamsVoiceApplications policy assigned. The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```
